frontend/cypress/integration/createTitleAndBooks.js

```diff
@@ -10,13 +10,24 @@ describe('This test will create a title and a book', () => {
   });
 
   it('Creates a title and reset the fields when added', () => {
-    cy.get('[data-cy=name]').type('Book made from Test');
-    cy.get('[data-cy=cost]').type('2912');
-    cy.get('[data-cy=isbn]').type('420-420');
+    const bookTitle = 'Book made from Test'
+    const isbn = '420-420'
+    const cost = '2912'
+
+    cy.get('[data-cy=name]').type(bookTitle);
+    cy.get('[data-cy=cost]').type(cost);
+    cy.get('[data-cy=isbn]').type(isbn);
     cy.get('[data-cy=titleType]').click({force:true});
     cy.get('div').contains('Kurslitteratur').click();
     cy.get('[data-cy=submit]').click();
+
+    cy.get('[data-jest=search]').type(bookTitle);
+    cy.get('[data-cy=table]').contains(bookTitle);
+
     cy.get('[data-cy=reset]').click();
+    cy.get('[data-cy=name]').should('have.not.value',bookTitle);
```

Comment:
Minor formatting comment
```suggestion
    cy.get('[data-cy=name]').should('have.not.value', bookTitle);
```